keyboards/pimoroni/keybow2040/readme.md

```diff
@@ -0,0 +1,101 @@
+    (not so great if you have configured BOOTSEL to do something)
+
+  * **Mr Resetti double-tap**: press RESET twice quickly
+    (only works when running the QMK firmware)
+
+Find the BOOTSEL and RESET buttons on your Keybow 2040 either side of
+the USB socket on the edge of the PCB. When the USB socket is facing
+away from you, BOOTSEL is on the right and RESET is on the left.
+According to the schematic the reset button is called Mr Resetti.
+
+  * _Aside_: The BOOTSEL button disables the QSPI flash that stores
+    the firmware, which makes the RP2040 use its built-in UF2
+    bootloader.
+
+A USB drive will appear called `RPI-RP2`. Copy the firmware file
+`pimoroni_keybow2040_default.uf2` onto the `RPI-RP2` drive using
+whatever software is convenient (e.g. the `cp` command, the macOS
+Finder, or Windows Explorer). The `make :flash` command will copy the
+firmware for you.
+
+The `RPI-RP2` drive will disappear (which might make your OS complain
+that you failed to unmount it first) and a keyboard device will appear
+in its place.
+
+The Pimoroni web site has [instructions for reinstalling CircuitPython
+on the Keybow 2040](https://learn.pimoroni.com/article/circuitpython-and-keybow-2040)
+
+
+notes
+-----
+
+  * The BOOTSEL button doubles as a 17th configurable key.
+
+  * The default keymap has F13 - F20 on the top half of the board
+    (nearest the USB socket), and a navigation cluster (arrows, home,
+    end, page up/down) on the bottom half. The BOOTSEL button changes
+    the RGB matrix mode.
+
+  * The Keybow 2040 CircuitPython firmware uses VID = 0x16D0, PID = 0x08C6,
+    but this QMK firmware has different USB IDs to avoid confusing drivers:
+    VID = 0x5069 ('Pi'), PID = 0x4784 (4 'x' 4)
+
+
+resources
+---------
+
+Here are some links to the information I used when configuring QMK for the Keybow 2040.
+
+  * [Piomoroni Keybow 2040 product page](https://shop.pimoroni.com/products/keybow-2040)
+  * [Keybow 2040 assembly instructions](https://learn.pimoroni.com/article/assembling-keybow-2040)
+
+Places to find information about the Keybow 2040 hardware:
+
+  * [schematic](https://cdn.shopify.com/s/files/1/0174/1800/files/keybow_2040_schematic.pdf)
+      * contains most of the information that can also be gleaned from the CircuitPython support
+
+  * Pimoroni user-friendly CircuitPython libraries
+      * [old](https://github.com/pimoroni/keybow2040-circuitpython) and [new](https://github.com/pimoroni/pmk-circuitpython)
+      * these are wrappers around the low-level support below
+
+  * [CircuitPython hardware support](https://github.com/adafruit/circuitpython/blob/main/ports/raspberrypi/boards/pimoroni_keybow2040/)
+      * defines map from hardware feature names (e.g. `SW15`) to the way they are accessed (e.g. GPIO6)
+
+  * [CircuitPython IS31FL3731 driver](https://github.com/adafruit/Adafruit_CircuitPython_IS31FL3731/blob/main/adafruit_is31fl3731/keybow2040.py)
+      * defines the RGB matrix layout
```

Comment:
This would be better linked in the "hardware supported" section above. Though, it looks like most of this is linked in the product page, already. 
```suggestion
```

Reply:
I have left these links in because it was difficult to find the relevant parts of the default Python firmware, so I think it will be helpful when anyone else wants to work on this board.

Reply:
But this isn't the circuit python repo, so has zero relevance here.  And most of the links add no value.